servers/draconitymc/metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+    "id": "draconitymc",
+    "name": "DraconityMC",
+    "website": "https://www.draconitymc.com",
+    "store": "https://store.draconitymc.com/",
+    "addresses": [
+        "draconitymc.com"
+    ],
+    "primaryAddress": "play.draconitymc.com",
+    "primaryColor": "#3399FF",
+    "secondaryColor": "#000000",
+    "minecraftVersions": [
+        "1.19.*",
+        "1.20.*"
+    ],
+    "primaryMinecraftVersion": "1.20.1",
+    "primaryRegion": "EU",
+    "regions": [
+        "AF",
+        "AS",
+        "EU",
+        "NA",
+        "OC",
+        "SA"
+    ],
+    "gameTypes": [
+        "Survival"
+    ],
+    "socials": {
+        "discord": "https://discord.com/invite/draconitymc",
+        "youtube": "https://www.youtube.com/@draconitymc"
```

Comment:
The `youtube` value should only include the slug as we prefix it with the appropriate URL location

Reply:
> The `youtube` value should only include the slug as we prefix it with the appropriate URL location

Hey, I have just removed the youtube link as I am not exactly sure what you meant, thanks!